RELEASE/scripts/autoscend/auto_buff.ash

```diff
@@ -111,7 +111,7 @@ boolean buffMaintain(effect buff, int mp_min, int casts, int turns, boolean spec
 	case $effect[Ashen]:					useItem = $item[pile of ashes];						break;
 	case $effect[Ashen Burps]:					useItem = $item[ash soda];						break;
 	case $effect[Astral Shell]:
-		if(acquireTotem())
+		if(have_skill($skill[Astral Shell]) && acquireTotem())
```

Comment:
use auto_have_skill instead of have_skill. Does check have_skill but also confirms it isn't path restricted. Replace in all instances below too